masakari-controller/controller/masakari_config.py

```diff
@@ -179,9 +267,15 @@ def _log_setup(self):
 
         level = self.conf_log.get('log_level')
 
+        logging_context_format_string = '%(asctime)s.%(msecs)03d %(process)d %(levelname)s ' \
+            + '%(name)s [%(thread_name)s %(record_identifier)s] %(message)s'
         logging.set_defaults(
+            logging_context_format_string=logging_context_format_string,
             default_log_levels=logging.get_default_log_levels() +
             ['controller=' + level])
+        RequestContext(thread_name=str(
```

Comment:
You want to hold thread name and resource id in the class, right? If so, I understand your purpose.

This RequestContext class needs variables related to keystone, defined in L70-83. I think it's always '-'.